core/src/index.ts

```diff
@@ -79,3 +79,4 @@ import "./config/tasks";
 import "./config/telemetry";
 import "./config/web";
 import "./config/websocket";
+import "./initializers/plugins";
```

Comment:
Not needed?

Reply:
Very needed. Plugins need this import to know the type for the api object.

Reply:
Ah ha!  Better than every plugin needing to require the initializer. 

To be complete, we probably should also import/export any other initializer that adds something to the global `api` object.  

* https://github.com/grouparoo/grouparoo/blob/main/core/src/initializers/apm.ts
* https://github.com/grouparoo/grouparoo/blob/main/core/src/initializers/codeConfig.ts
* https://github.com/grouparoo/grouparoo/blob/main/core/src/initializers/next.ts
* https://github.com/grouparoo/grouparoo/blob/main/core/src/initializers/notifiers.ts
* https://github.com/grouparoo/grouparoo/blob/main/core/src/initializers/plugins.ts
* https://github.com/grouparoo/grouparoo/blob/main/core/src/initializers/session.ts